src/lib/blueprint.ts

```diff
@@ -2,6 +2,24 @@ import type { Openapi } from './openapi.js'
 
 export interface Blueprint {
   name: string
+  language: string
+  methodDescription: string
+  versionInformation: string
```

Comment:
I think `versionInformation` is specific to SDK versioning so probably it shouldn't be part of the blueprint